### ISSUE TITLE
Durations when server crashes

### ISSUE THREAD

**Issue by pansk** [NVIDIA/DIGITS#771]
During the weekend my machine has been unplugged, now the jobs that was running/waiting are reporting an execution time of 3 days and the 0 secs respectively, and from the dates they look like they were aborted when I re-launched the server (the two training jobs i had put in queue on Friday were supposed to last just few hours, so probably the machine was accidentally turned off on saturday).

Is it possible, at least for the running jobs, to use another date/time as "abort time", e.g. the timestamp for the last valid batch?

**Comment by lukeyeager** [comment]
Can you provide a screenshot of what you're talking about? It'd be useful to see the times reported on the homepage, as well as the detailed timing info for each job.

**Comment by pansk** [comment]
This is how the two processes are listed in the homepage:

![image](https://cloud.githubusercontent.com/assets/1263535/15540736/55274e24-2289-11e6-8055-029c44694e4b.png)

**Comment by pansk** [comment]
This was the running process:

![image](https://cloud.githubusercontent.com/assets/1263535/15540848/d4554200-2289-11e6-9983-91a03e594801.png)

The digits process was launched just after boot, which from logs started at May 23, 1:54PM

**Comment by pansk** [comment]
And this was the other process in queue, which never started:

![image](https://cloud.githubusercontent.com/assets/1263535/15540923/2e7046cc-228a-11e6-8717-e2341d1ce6aa.png)

Similar processes (with SGD in place of RMSProp) normally took <20h (I didn't run the same process because from what the ~50% of the process that ran, it wasn't really promising. If you really need a proper estimate of how long should have taken, I could let it run a couple of epochs and see the projected duration (but I'm pretty sure it's not going to be 6 days).

**Comment by lukeyeager** [comment]
Ok I see what you're saying. It's reporting "0s" on the homepage because the second job was stuck waiting and never actually started running its one task.

IIRC, this all started with your suggestion in https://github.com/NVIDIA/DIGITS/issues/706. I like the new behavior better than the old behavior, even if it's confusing in this case. I feel like this particular situation is pretty rare.

**Comment by pansk** [comment]
Yep, 0s is clear (the task actually never ran), the more confusing is the other one, for which "3 days" of runtime is a pretty unexpected value (it should have been in the order of hours). Seems like the server actually writes the "abort" during next restart.
Of course this is rare and quite unimportant (and I don't expect a fix), but if (my hypothesis) DIGITS is already writing a timestamp during training, using the last one of those timestamps as "abort time" would result in a better approximation, with a (relatively) easy fix.
